autotest/gcore/numpy_rw.py

```diff
@@ -257,9 +257,9 @@ def test_numpy_rw_10_bis(options):
 # Test all datatypes
 
 
-def test_numpy_rw_11():
-
-    type_tuples = [
+@pytest.mark.parametrize(
+    "name,in_dt,np_dt,val",
+    [
         ("uint8", gdal.GDT_Byte, numpy.uint8, 255),
```

Comment:
Was there a reason why the `np.bool_` test was excluded?

Reply:
> Was there a reason why the `np.bool_` test was excluded?

As it was written, it didn't pass, given creating a GDAL raster with GDT_Byte returns uint8, not bool.